modules/swagger-codegen/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -990,6 +990,16 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/ReadOnlyFirst'
+  Configuration:
+    type: object
+    properties:
+      configuration:
+        type: string
+  Client:
```

Comment:
I think this model needs to have an associated endpoint, since the clashing issue is in the *Api types.